core/kazoo_voicemail/src/kz_vm_message.erl

```diff
@@ -43,6 +43,7 @@
 -define(MODB_COUNT_VIEW, <<"mailbox_messages/count_per_folder">>).
 -define(COUNT_BY_VMBOX, <<"mailbox_messages/count_by_vmbox">>).
 -define(BOX_MESSAGES_CB_LIST, <<"vmboxes/crossbar_listing">>).
+-define(PVT_LEGACY_TYPE, <<"private_media">>).
```

Comment:
really ?
[check here](https://github.com/2600hz/kazoo/blob/master/core/kazoo_documents/src/kzd_callflow.erl#L22)
[or here](https://github.com/2600hz/kazoo/blob/master/core/kazoo_documents/src/kz_notification.erl#L48)
or any other in kazoo_documents